### ISSUE TITLE
Golems not working if the player is too far away

### ISSUE THREAD

**Issue by Mistercl0ud** [Azanor/thaumcraft#1655]
The golems don't seem to do anything if you are too far away from them even if you use a chunkloader on a single player game or if you use Forge Essential ( in singleplayer ). 
The rest work perfectly fine like the advanced alchemical furnace or the hungry chest. Other mods don't have that problem neither ( Forestry, IndustrialCraft, Railcraft ).

**Comment by Azanor** [comment]
MC is a bit goofy about how it handles entities near unloaded chunks. All chunks within 32 blocks of an entity needs to be loaded for it to update it's ticks - where all those chunks loaded in your test?

**Comment by Mistercl0ud** [comment]
Yes they were all loaded including chunks around. The fisherman golem seems to be unaffected by this problem.  The one that don't work properly are the storing core and the harvest core.

**Comment by doushio9967** [comment]
Are you using optifine?
It tends to be the cause for far-away (but same dimension) chunkloaded golems stopping, in SSP.
For that problem someone wrote a small mod, optinerf, to undo optifine's rather aggressive canSkipEntityUpdate and make golems work again.

Might be worth trying without optifine, assuming you're using it, to see if that's the cause.

**Comment by Chefbarbie** [comment]
@doushio9967 
Where is this mod located? I can't find it with google.

**Comment by GirafiStudios** [comment]
@Chefbarbie It's literally the first site which shows up, when you Google Optifine. Here's a link in case you could not find it for some reason: https://optifine.net/home

**Comment by Chefbarbie** [comment]
@GirafiStudios 
Thanks for the effort but it's optifine causing this problem. The mod I tried to google is "optinerf", mentioned in the post I addressed.

**Comment by doushio9967** [comment]
@Chefbarbie 
On the jp minecraft forums.
http://forum.minecraftuser.jp/viewtopic.php?f=13&t=2150 , under the おまけ section.
Surprised it doesn't show up on google. Alltho the author's tweet on it does show up..

**Comment by Chefbarbie** [comment]
Well serves me for not knowing how to read Japanese.

**Comment by Mistercl0ud** [comment]
Optinerf : http://forum.minecraftuser.jp/download/file.php?id=53245
